src/client/method_list_units.c

```diff
@@ -0,0 +1,151 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+#include "libhirte/common/string-util.h"
+
+#include "method_list_units.h"
+
+static int
```

Comment:
This looks like a format error?

Reply:
It looks like that, but its really formatted via `clang-format`.